examples/standalone-mounted/main.tf

```
@@ -0,0 +1,63 @@
+provider "aws" {
+  assume_role {
+    role_arn = var.aws_role_arn
+  }
+
+  default_tags {
+    tags = var.tags
+  }
+}
+
+# Keypair for SSH
+# ---------------
+resource "tls_private_key" "main" {
+  algorithm = "RSA"
+}
+
+resource "local_file" "private_key_pem" {
+  filename = "${path.module}/work/private-key.pem"
+
+  content         = tls_private_key.main.private_key_pem
+  file_permission = "0600"
+}
+
+resource "aws_key_pair" "main" {
+  public_key = tls_private_key.main.public_key_openssh
+
+  key_name = "${var.friendly_name_prefix}-ssh"
+}
+
+# Store TFE License as secret
+# ---------------------------
+
+module "secrets" {
+  source = "../../fixtures/secrets"
+  tfe_license = {
+    name = "tfe-license"
+    path = var.license_file
+  }
+}
+# Standalone, mounted disk
+# ------------------------
+module "standalone" {
+  source = "../../"
+
+  operational_mode    = "disk"
+  acm_certificate_arn = var.acm_certificate_arn
```

```suggestion
  disk_path           = "/opt/hashicorp/data"
  acm_certificate_arn = var.acm_certificate_arn
```

I know that the value is already defaulted, but because this is a mounted disk example, I think it's good to have the variable defined here.

---

added